pkg/ovsdb/client/client.go

```diff
@@ -25,7 +25,9 @@ const (
 	ICNBDB = "icnbdb"
 	ICSBDB = "icsbdb"
 )
-const timeout = 3 * time.Second
+
+const connectTimeout time.Duration = time.Second * 20
+const inactivityTimeout time.Duration = time.Second * 180
```

Comment:
I think we should make the timeout configurable.

Reply:
done